strategy/measuring-impact.md

```diff
@@ -0,0 +1,148 @@
+# Human Cell Atlas Data Coordination Platform - Our Strategy
+
+## Measuring our Impact
+
+This document provides background context for the overall aims and the specific aims, including:
+* ***value assumptions*** that define why we believe our aims are the right aims, and
+* ***indicators for success*** that define how we assess our progress against these aims
+
+### Aim 1
+
+**Create a data resource that maximizes the value and use of Human Cell Atlas data across the scientific community.**
+
+#### Value Assumptions
+
+Scientists expect the DCP to be a comprehensive resource, containing all data that has been generated in their area of research interest. If they aren’t confident that all the data they need is present, if metadata they need to interpret the data are missing, or if the data quality is too low, they will download HCA data from other resources.
+
+Scientists will use raw data from the DCP and “output data” from DCP processing pipelines as long as it is well described, and includes rich, detailed information about how the data was generated (including sample descriptions and protocol information).
+
+HCA metadata and data will generate significant scientific impact for a variety of researchers (namely the spectrum from computational biologists to wet-lab biologists) if represented in an accessible manner according to their technical needs and capabilities.
+
+Data generators will deposit data in the DCP because the DCP provides valuable incentives, such as accession numbers that can be cited in publications.
```

Comment:
This seems to be a weak carrot.  Assuming a researcher's funding is not mandating submitting to the HCA DCP then what motivation do they really have?  Providing accessions does not seem like it's the most critical feature.  As a researcher I'd like to hear I should upload to the HCA because 1) we pay for storage on the cloud, 2) we analyze the data with the best pipelines, 3) I can compare my data with other related datasets in the project, 4) maybe I can get early access to my analyzed data to check it before it's on the full portal.

I think these are powerful reasons someone would want to submit (or may want to submit in the future).  But we don't really communicate this well to researchers.  And I don't think we can do this comprehensively now which hurts our case of asking for submissions.